specification/cognitiveservices/OpenAI.Inference/models/completions/extensions/oyd/vector_search.tsp

```diff
@@ -53,8 +53,10 @@ model OnYourDataEndpointVectorizationSource
   @doc("Specifies the resource endpoint URL from which embeddings should be retrieved. It should be in the format of https://YOUR_RESOURCE_NAME.openai.azure.com/openai/deployments/YOUR_DEPLOYMENT_NAME/embeddings. The api-version query parameter is not allowed.")
   endpoint: url;
 
+  // `authentication` according to the 05-01-preview spec, only supports api-key and data access token auth.
   @doc("Specifies the authentication options to use when retrieving embeddings from the specified endpoint.")
-  authentication: OnYourDataAuthenticationOptions;
+  @added(ServiceApiVersions.v2024_05_01_Preview)
+  authentication: OnYourDataVectorSearchAuthenticationOptions;
```

Comment:
Is this a known breaking change?

OnYourDataAuthenticationOptions -> OnYourDataVectorSearchAuthenticationOptions

Reply:
There was a changed in `v05_01`narrowing down the supported authentication types:

https://github.com/Azure/azure-rest-api-specs/blob/162fb3af1af3a6e166ab16b8f2684c31a6e6bd0b/specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2024-05-01-preview/inference.yaml#L3353-L3375